README.md

```diff
@@ -1,4 +1,21 @@
 # VIP Go / WordPress.com Compatibility
 
-This plugin provides compatibility for sites that are moving from WordPress.com
-to VIP Go.
+This plugin provides compatibility for sites that are moving from WordPress.com to VIP Go.
+
+## Deprecated Functions
+
+The following functions are deprecated on VIP Go and are added as shims to keep themes and plugins from throwing fatal errors:
+
+* `wpcom_vip_load_wp_rest_api()` - Loads the built-in WP REST API endpoints in WordPress.com VIP context.
```

Comment:
Should people continue to use this? Or switch to a different function? If we are deprecating something, we should indicate the path forward to get people to stop using it. :)

Reply:
Good point!  It just needs removed completely. I'll update :)